website/views.py

```diff
@@ -39,8 +40,13 @@ def dispatcher(request, permalink=''):
         'navs': blocks['navs'],
         'sidebar': blocks['sidebar'],
         'footer': blocks['footer'],
-        'permalink': permalink,
+        'permalink': permalink
     }
+    if not get_rss():
+        context['rssfeed'] = [('Click here for latest news from python.org',
+                               'https://pythoninsider.blogspot.com/')]
+    else:
+        context['rssfeed'] = get_rss()
```

Comment:
function should be called only once.
the function will return the default value if no RSS.

Reply:
updated.